Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst

```diff
@@ -0,0 +1,159 @@
+.. |author| replace:: *Tiago Ribeiro*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Michael Reuter*
+
+.. _Observing-Interface-Setup-DDS-Daemon-in-Nublado:
+
+###########################
+Setup DDS Daemon in Nublado
+###########################
+
+.. _Setup-DDS-Daemon-in-Nublado-Overview:
+
+Overview
+========
+
+When operating in nublado on one of the :ref:`operational environments <Observing-Interface-Operational-Environments>`, users are required to start a DDS daemons.
+The DDS daemon is in charge of handling most of the DDS communication and considerably alleviates the load on users notebooks and processes in general.
+
+
+.. _Setup-DDS-Daemon-in-Nublado-Prerequisites:
+
+Prerequisites
+=============
+
+- You must be logged into the nublado system in one of the :ref:`operational environments <Observing-Interface-Operational-Environments>`.
+- See :ref:`Observing-Interface-Getting-Started-Nublado`, if you need help getting started.
+
+.. _Setup-DDS-Daemon-in-Nublado-Post-Conditions:
+
+Post-Condition
+==============
+
+
+- Users can reliably communicate with the observatory system using DDS daemon.
+
+.. _Setup-DDS-Daemon-in-Nublado-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Once you are logged to nublado, open a "terminal tab".
+If you are in the :ref:`Launcher <Observing-Interface-Getting-Started-Nublado-JupyterLab>` view, click on the "Terminal" icon in the "Other" tab at the bottom.
+
+If you don't see the :ref:`Launcher <Observing-Interface-Getting-Started-Nublado-JupyterLab>` view, click on the plus sign at the top-left hand side of the view, as indicated in the :ref:`figure bellow <Setup-DDS-Daemon-in-Nublado-New-Launcher>`.
+
+.. figure:: ./_static/nublado-jupyter-lab-new-launcher.png
+    :name: Observing-Interface-Getting-Started-Nublado-JupyterLab
+
+    Screenshot of jupyter-lab section highlighting the "New Launcher" button.
+
+The procedure is divided in two steps; :ref:`Setup-DDS-Daemon-in-Nublado-Procedure-Steps-Setup-and-Start-DDS-Daemon` and :ref:`Setup-DDS-Daemon-in-Nublado-Procedure-Steps-Nublado-User-Setup`.
+
+.. _Setup-DDS-Daemon-in-Nublado-Procedure-Steps-Setup-and-Start-DDS-Daemon:
+
+Setup and Start DDS Daemon
+--------------------------
+
+.. note::
+
+	This step needs to be performed every time the user nublado instance is restarted, or if the user shut down the daemon.
+
+In the terminal enter the following sequence of commands to setup your environment;
+
+.. prompt:: bash
+
+  source ${LOADSTACK}
+  export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
+  source $OSPL_HOME/release.com
+  export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
```

Comment:
This step returned an error for me:

`$ export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'lsst'
`